signer-service/src/api/services/pendulum.service.js

```diff
@@ -56,7 +62,32 @@ exports.sendStatusWithPk = async () => {
   const { fundingAccountKeypair, fundingAmountRaw } = getFundingData(apiData.ss58Format, apiData.decimals);
   const { data: balance } = await apiData.api.query.system.account(fundingAccountKeypair.address);
 
-  if (Big(balance.free.toString()).gte(fundingAmountRaw)) {
+  const tokensToCheck = Object.keys(TOKEN_CONFIG);
+  let isTokensSufficient = true;
+
+  // Wait for all required token balances check.
+  await Promise.all(
+    tokensToCheck.map(async (token) => {
+      const tokenConfig = TOKEN_CONFIG[token];
+      console.log(`Checking token ${token} balance...`);
+      const tokenBalanceResponse = await apiData.api.query.tokens.accounts(
+        fundingAccountKeypair.address,
+        tokenConfig.pendulumCurrencyId,
+      );
+
+      const tokenBalance = Big(tokenBalanceResponse?.free?.toString() ?? '0');
+      const maximumSubsidyAmountRaw = Big(tokenConfig.maximumSubsidyAmountRaw);
+      const subsidyRatio = tokenBalance.div(maximumSubsidyAmountRaw);
```

Comment:
Maybe rename this and all related variables to something like `remainingSubsidiesAvailableCount` or similar. I think the word ratio is a bit misleading.

Reply:
True, quite confusing after I look again at it.

Reply:
I'll leave `SUBSIDY_MINIMUM_RATIO_FUND_UNITS` if that makes sense, since it is the actual ratio of "minimum amount of subsidies considering worse case scenario". Quite hard to find a name for it !